addons/dragging/functions/fnc_dropObject_carry.sqf

```diff
@@ -46,6 +46,7 @@ if (_tryLoad && {!(_target isKindOf "CAManBase")} && {["ace_cargo"] call EFUNC(c
 } else {
     // Release object
     detach _target;
+    [QEGVAR(common,setVelocity),[_target,getVelocity _unit],_target] call CBA_fnc_targetEvent;
```

Comment:
spaces between commas and parameters pls

Reply:
should be done meow